sdk/monitor/Azure.Monitor.OpenTelemetry/api/Azure.Monitor.OpenTelemetry.netstandard2.0.cs

```diff
@@ -0,0 +1,17 @@
+namespace Azure.Monitor.OpenTelemetry
+{
+    public static partial class AzureMonitorOpenTelemetryExtensions
+    {
+        public static Microsoft.Extensions.DependencyInjection.IServiceCollection AddAzureMonitorOpenTelemetry(this Microsoft.Extensions.DependencyInjection.IServiceCollection services, Azure.Monitor.OpenTelemetry.AzureMonitorOpenTelemetryOptions options) { throw null; }
+        public static Microsoft.Extensions.DependencyInjection.IServiceCollection AddAzureMonitorOpenTelemetry(this Microsoft.Extensions.DependencyInjection.IServiceCollection services, Microsoft.Extensions.Configuration.IConfiguration configuration) { throw null; }
+        public static Microsoft.Extensions.DependencyInjection.IServiceCollection AddAzureMonitorOpenTelemetry(this Microsoft.Extensions.DependencyInjection.IServiceCollection services, System.Action<Azure.Monitor.OpenTelemetry.AzureMonitorOpenTelemetryOptions> configureAzureMonitorOpenTelemetry, string? name = null) { throw null; }
+        public static Microsoft.Extensions.DependencyInjection.IServiceCollection AddAzureMonitorOpenTelemetry(this Microsoft.Extensions.DependencyInjection.IServiceCollection services, bool enableTraces = true, bool enableMetrics = true, bool enableLogs = true) { throw null; }
```

Comment:
I would say remove this version. Nice convenience, but the default parameters will make it really hard expand in the future. If you want to see, drop in another method with some defaults like...

```csharp
    public static OpenTelemetryLoggerOptions AddOneCollectorExporter(bool enableMetrics = false, bool enableTraces = false, bool enabledLogs = false)
    {
    }

    public static OpenTelemetryLoggerOptions AddOneCollectorExporter(Action<OneCollectorLogExporterOptions> configure, string? name = null)
    {
    }
```

You should get a warning RS0027 which links to: https://github.com/dotnet/roslyn/blob/main/docs/Adding%20Optional%20Parameters%20in%20Public%20API.md

In the OTel repos we have moved away from using default parameters in extensions, probably most places in public API surface.